lib/Time/gmtime.pm

```diff
@@ -34,8 +28,8 @@ sub populate (@) {
     return $tmob;
 } 
 
-sub gmtime (;$)    { populate CORE::gmtime(@_ ? shift : time)}
-sub gmctime (;$)   { scalar   CORE::gmtime(@_ ? shift : time)} 
+sub gmtime  :prototype(;$) { populate CORE::gmtime(@_ ? shift : time) }
+sub gmctime :prototype(;$) { scalar   CORE::gmtime(@_ ? shift : time) }
 
 1;
```

Comment:
We can also drop all the `1;` lines now that we have the `5.38` feature bundle with `module_true`.

Reply:
good point, will fix